enioka_scan/src/main/java/com/enioka/scanner/sdk/bluebird/BluebirdScanner.java

```diff
@@ -63,6 +63,21 @@ public void onReceive(Context context, Intent intent) {
     }
 
 
+    ////////////////////////////////////////////////////////////////////////////////////////////////
+    // SOFTWARE TRIGGERS
+    ////////////////////////////////////////////////////////////////////////////////////////////////
+
+    @Override
+    public void pressScanTrigger() {
```

Comment:
you could actually add a default method to the interface rather than modifying all SDKs.